metrics/mixin/alerts/perf.libsonnet

```diff
@@ -0,0 +1,26 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'ODF-ceph-mds-high-cpu-warnings.rules',
+        rules: [
+          {
+            alert: 'MDS-high-cpu',
+            expr: |||
+              pod:container_cpu_usage:sum{%(mdsSelector)s}/ on(pod) kube_pod_resource_limit{resource='cpu',%(mdsSelector)s} > 0.67
```

Comment:
`Question:` Why 67%? Is it a randomly selected or is there a reason for it?

Reply:
Yes, by default the MDS pods are allocated 3 CPU. 67% means 2 CPUs are busy. I have given 6 hours because if there is some intermittent usage of CPU, like some scan job that reads every file or something, then they don't need multiple active MDS, if it is persistently high, then we should alert. That is the intent.